test/unit/math/prim/core/init_threadpool_tbb_test.cpp

```diff
@@ -15,7 +15,6 @@ TEST(intel_tbb_new_init, check_status) {
   EXPECT_EQ(1, tbb_init.max_concurrency());
 #endif
 
-  auto& tbb_reinit = stan::math::init_threadpool_tbb();
```

Comment:
This seems like it's itentional, since that function is side-effecting. At least, if this still passes, I don't understand the threadpool as well as I think I do.

Reply:
Hmm good point, will just revert